Modules/PolkadotMinimalModule.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types = 1);
+
+/*  Copyright (c) 2023 Nikita Zhavoronkov, nikzh@nikzh.com
+ *  Copyright (c) 2023 3xpl developers, 3@3xpl.com
+ *  Distributed under the MIT software license, see the accompanying file LICENSE.md  */
+
+/*  This is a module for Polkadot. See PolkadotLikeMinimalModule.php for details.  */
+
+final class PolkadotMinimalModule extends PolkadotLikeMinimalModule implements Module
+{
+    function initialize()
+    {
+        // CoreModule
+        $this->blockchain = 'polkadot';
+        $this->module = 'polkadot-minimal';
+        $this->is_main = true;
+        $this->currency = 'polkadot';
+        $this->currency_details = ['name' => 'DOT', 'symbol' => 'DOT', 'decimals' => 10, 'description' => null];
```

Comment:
It is better to keep a comment about [redenomination](https://wiki.polkadot.network/docs/learn-redenomination) on block 1 248 328